frontend/dashboard/pages/PageLayout/DashboardHeader/SmallHeaderMenu/SmallHeaderMenuItem/SmallHeaderMenuItem.tsx

```diff
@@ -16,8 +15,7 @@ export const SmallHeaderMenuItem = ({
   onClick,
 }: SmallHeaderMenuItemProps): ReactElement => {
   const { t } = useTranslation();
-  const location = useLocation();
-  const currentRoutePath: string = UrlUtils.extractSecondLastRouterParam(location.pathname);
+  const origin = window.location.origin;
```

Comment:
Because we have added `/dashboard/app-dashboard` to the path, the NavLink redirects by only removing the string after the last `/`. This means that the path to Gitea becomes wrong, and we should only have `basePath/itemHref`